lib/engines/hcktest/hcktest.rb

```diff
@@ -147,13 +147,25 @@ def run_clients(scope, run_opts = {})
                                 this platform is incorrect'
     end
 
+    def run_clients_post_start_host_commands
+      @project.engine.drivers&.each do |driver|
+        driver.post_start_commands&.each do |command|
+          return unless command.host_run
+
+          @logger.info("Running command (#{command.desc}) on host")
+          run_cmd(command.host_run)
+        end
+      end
+    end
+
     def configure_and_synchronize_clients
       run_only = @project.options.test.manual && @project.options.test.driver_path.nil?
 
       @clients.each_value do |client|
         client.configure(run_only:)
       end
 
+      run_clients_post_start_host_commands
```

Comment:
Why is this change in `HCKTest` while `guest_run` is handled in `HCKClient`?

Reply:
Because of this comment  https://github.com/HCK-CI/AutoHCK/pull/478#discussion_r1785852107